filter_linux.go

```diff
@@ -172,6 +172,14 @@ func (h *Handle) FilterAdd(filter Filter) error {
 		if filter.ClassId != 0 {
 			nl.NewRtAttrChild(options, nl.TCA_U32_CLASSID, nl.Uint32Attr(filter.ClassId))
 		}
+		if filter.Divisor != 0 {
+			// Validate if the divisor is a power of two.
```

Comment:
Pls remove this comment, the error message is explains it all

Reply:
Fixed.